Makefile

```diff
@@ -25,6 +25,16 @@ json-schema.tar.gz: $(OUT)
 	tar -czf json-schema.tar.gz --exclude '.*' json-schema
 	rm -rf json-schema
 
+venv: $(VENV)
+
+$(VENV):
+	python -m venv .venv
+	. .venv/bin/activate && python -m pip install --upgrade pip setuptools wheel
```

Comment:
What are you trying to achieve with this? Upgrading setuptools and wheel shouldn't be necessary.

Reply:
@webknjaz copy-paste error from some notes for a project where I needed to uses these, thanks for catching!